docs/getting-started.md

```diff
@@ -27,7 +27,7 @@ For the full list of supported package managers and Version Control Systems see
 
 ## 2. Download & Install ORT
 
-In future we will provide binaries of the ORT tools, but currently you have to build the tools on your own. First
+In future, we will provide binaries of the ORT tools, but currently you have to build the tools on your own. First
```

Comment:
While at it, maybe yet better "In *the* future, ..."? Also, we usually use passive voice in docs and should not talk about "we" at all. Thinking about it, I'd actually drop the first sentence completely and start saying "You currently have to build the tools on your own". However, that's also not quite correct, as you could use the CLI fat JAR from JitPack...